addons/event/static/src/views/event_registration_kanban_controller.js

```diff
@@ -0,0 +1,43 @@
+import { kanbanView } from "@web/views/kanban/kanban_view";
+import { KanbanController } from "@web/views/kanban/kanban_controller";
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+export class EventRegistrationKanbanController extends KanbanController {
+
+    setup() {
+        super.setup()
+        this.dialog = useService("dialog");
+        this.orm = useService("orm");
+    }
+
+    async openRecord(record, mode) {
+        if (this.props.context.is_registration_desk_view) {
```

Comment:
Not sure why this code end up in a kanban controller... shouldn't we handle it in ``onClickSelectAttendee`` ? Seems in the end it had nothing to do with kanban code ?

Reply:
Oh right, indeed you are already on the view. What I don't like is that you rely on context, have to say this is a bit hackish. I think it would be better to use dedicated views (event_registration_action_frombarcode) which would always have this behavior (aka: calling that view instead of the old one with a context key). However this cannot be done in 17.4 as upgrade would not be happy. So ok for stable, and maybe for master create dedicated actions ?